test/testData.js

```diff
@@ -325,6 +325,11 @@ var testPoints = [
     ll: [2, 1],
     xy: [1057002.4052152266, 110955.14117382761]
   },
+  {
+    code: '+proj=utm +lon_0=-3 +ellps=GRS80 +lat_1=0.5 +lat_2=2 +n=0.5',

```

Comment:
Hmm, I just run some test code and it looks like this is how it is supposed to be. Using `+lon_0=-3` you set in what is supposed to be `zone=30`, but is calculated to be `zone=29`. The same happens with `+proj=utm +zone=33 +datum=WGS84 +units=m +no_defs`, we specify `zone=33`, but it will be calculated to be `zone=32`.